server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -1308,7 +1308,7 @@ private String getNewTsFileName(long timePartitionId) {
   }
 
   private String getNewTsFileName(long time, long version, int mergeCnt) {
```

Comment:
add a parameter

Reply:
change method parameter to (time, version, mergeCnt, unSeqMergeCnt)